index.d.ts

```diff
@@ -457,4 +447,23 @@ declare module 'gqutils' {
 		arg: (arg: any, opts?: string[] | {pick?: string[]}) => string;
 		toGqlArg: typeof toGqlArg;
 	}
+
+	class GqlApi<FragmentsMap = gqlFragmentMap, EnumsMap = gqlEnumMap> extends Gql<gqlFragmentMap, gqlEnumMap> {
```

Comment:
Types passed should not be default but the input
```suggestion
	class GqlApi<FragmentsMap = gqlFragmentMap, EnumsMap = gqlEnumMap> extends Gql<FragmentsMap, EnumsMap> {
```